integration-tests/src/test/kotlin/org/stellar/anchor/platform/suite/AbstractIntegrationTests.kt

```diff
@@ -24,6 +32,19 @@ abstract class AbstractIntegrationTests(val config: TestConfig) {
   var walletKeyPair = SigningKeyPair.fromSecret(CLIENT_WALLET_SECRET)
   var anchor = wallet.anchor(config.env["anchor.domain"]!!)
   var token: AuthToken
+  private val submissionLock = Mutex()
+
+  suspend fun transactionWithRetry(transactionLogic: suspend () -> Unit) =
```

Comment:
Nice function!

:nit: Can we add parameters to the function?

```suggestion
  suspend fun transactionWithRetry(retries=5, range=5, transactionLogic: suspend () -> Unit) =
```

Reply:
Should we name the function with a more generic name?

Reply:
I think the naming makes sense, it's only meant to be used for submitting Stellar transactions because it also holds a lock. We should not be using it for anything else.

Reply:
ok, we can rename when we have other use cases.